### PR TITLE
feat(deploy/ha): Add halconfig yamls for clouddriver-ro, clouddriver-rw, and clouddriver-caching.

### PR DESCRIPTION
**Pull request by joonlim**

### CODE REVIEW

**Comment by joonlim** [comment]
@jtk54 Please review this PR when possible

File: halconfig/clouddriver-caching.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# halconfig
+
+admin.tasks.shutdownWaitSeconds: 600 # 10 minutes
```
Comment (lwander) [inline halconfig/clouddriver-caching.yml]: Is there a reason not to rely on the base clouddriver.yml for these shared properties?

**Comment by lwander** [comment]
Nice, LGTM

**Comment by joonlim** [comment]
@lwander This one's also ready to be merged please

**Pull request merged**